akka-http-bench-jmh/src/main/scala/akka/http/impl/model/parser/UriParserBenchmark.scala

```diff
@@ -0,0 +1,35 @@
+package akka.http.impl.model.parser
+
+import java.util.concurrent.TimeUnit
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.Uri
+import akka.parboiled2.UTF8
+import org.openjdk.jmh.annotations._
+import org.openjdk.jmh.infra.Blackhole
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+@State(Scope.Benchmark)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@BenchmarkMode(Array(Mode.Throughput))
+class UriParserBenchmark {
+  implicit val system: ActorSystem = ActorSystem("uri-parser-benchmark")
```

Comment:
Is the system really needed?

Reply:
That is right, not sure why I left it there